include/MassUpdate.php

```diff
@@ -1332,7 +1332,7 @@ public function doMassUpdateFieldsExistForFocus()
                     case "account_name":
                     case "bool":
                     case "enum":
-					case "dynamicenum":
+		    case "dynamicenum":
```

Comment:
seems way off may have to fix the indenting around it, e.g for the rest of the switch statement